### ISSUE TITLE
CAMARA QoD - Planning for release v0.9.0

### ISSUE THREAD

**Issue by hdamker** [camaraproject/QualityOnDemand#136]
Within the last CAMARA QoD Call we discussed to work towards a release of v0.9.0 of the QoD repository within June.
 
This issue is meant to discuss the time plan.
 
Here is the proposed plan for that (calculated backwards) – all dates are QoD Community Calls. 
 
### April 21st

- Freeze of content for release v0.9.0
- All contributions for v0.9.0 must exists either as PR or already discussed within an issue (in case of smaller changes)

### May 5th

- All PRs for v0.9.0 at least ready for FINAL review

### May 19th

- Final decision which PRs will be merged for v0.9.0 (if not already done)
- Creation of prerelease of v0.9.0  (v.0.9.0-rc1, "Release Candidate")
- Purpose of the release candidate is that implementations can be started in order to identify bugs or other implementation obstacles

### June 2nd

- Review of the prerelease, agree on bug fixes which came up, potentially revert features which have identified as not mature

### June 16th

- Release ready for final review (potentially rc2)

### June 23rd (with backup June 30th)

- Release of v0.9.0 of CAMARA QoD sub project (API definition and test cases)
- Updates of provider implementations will follow later.

 
In parallel the discussion of release content for release v0.10.0 can already start, which means to work on issues / concepts - release content to be decided on June 30th (with a target to release within September)

**Comment by hdamker** [comment]
@eric-murray @RandyLevensalor @jlurien @emil-cheung As the main contributors for the remaining issues for v0.9.0 please have a look on the plan above and comment.

**Comment by RandyLevensalor** [comment]
> @eric-murray @RandyLevensalor @jlurien @emil-cheung As the main contributors for the remaining issues for v0.9.0 please have a look on the plan above and comment.

@hdamker I can have  a PR submitted next week to meet this scheduler.  I think that we are down to fine tuning the names and descriptions of the attributes for the QoS Profiles.  We should be able to resolve these open topics or at least have a vote to decide at the next meeting.
